### ISSUE TITLE
I can't watch both a file and a directory

### ISSUE THREAD

**Issue by julienw** [remy/nodemon#1640]
- `nodemon -v`: 2.0.1
- `node -v`: 10.16.3
- Operating system/terminal environment: Linux Debian / gnome-terminal
- Using Docker? What image:  no
- Command you ran:
```
nodemon -w local-config.json -w src --exec echo true
```
### Expected behaviour
When local-config.json changes, the server is restarted.

### Actual behaviour
That doesn't work for local-config.json (but works for src/**)

This works fine if I don't specify  `-w src` though, so it's possible to watch that file just fine. Just it doesn't work when I specify both.

### Steps to reproduce
Have both a local file and a directory to watch.
(Note: I didn't try for other files)

Thanks!

**Comment by julienw** [comment]
I'm pretty sure this is still happening! :-)

**Comment by remy** [comment]
I'm trying to lay off the machine over the holiday period, but I've labeled this so that stalebot won't try to close it. Testing on a mac it behaves as expected.

Can you also try with the current version of nodemon (2.0.2 - though I suspect the OS + chokidar to be the source of the issue, but it's worth ruling out).

**Comment by julienw** [comment]
yeah, I have the same problem with 2.0.2.

chokidar seems to use a special API on MacOS X so I'm not surprised the behavior is different indeed.
I can try to write a testcase using chokidar only...

**Comment by julienw** [comment]
AH, this is interesting. I tried using chokidar, and ran the code with various versions of it, and it always worked. THEN I tried again with nodemon, and now it works...
The version of chokidar used with nodemon is v3.3.0 right now, but I don't know if that was changed when I tested...
Last time I had a similar problem this was a problem with transitive dependencies versions. Maybe upgrading chokidar to the latest version in package.json would fix the issue for good?

At the moment I can't reproduce the problem with latest nodemon...
